.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -72,7 +72,8 @@ phases:
           -e SIG_IMAGE_NAME=${SIG_IMAGE_NAME} \
           -e SIG_IMAGE_VERSION=${SIG_IMAGE_VERSION} \
           -e TELEPORTD_PLUGIN_DOWNLOAD_URL=${TELEPORTD_PLUGIN_DOWNLOAD_URL} \
-          -e IMPORT_IMAGE_URL=${IMPORT_IMAGE_URL} \
+          -e IMPORT_IMAGE_URL_GEN1=${IMPORT_IMAGE_URL_GEN1} \
+          -e IMPORT_IMAGE_URL_GEN2=${IMPORT_IMAGE_URL_GEN2} \
```

Comment:
I'm curious why we cannot use one variable and let the pipeline pass in the right value? This builder.yaml only needs to build one VHD at one time, so it doesn't need to know both gen2 and gen1 url, right?

Reply:
As I understand it we've got one set of pipeline variables that applies to all of the jobs in the run. If there's a clean way to target the same variable with different values in two different jobs from the same pipeline run I'd be happy to refactor in the future.